src/gleam/string.gleam

```diff
@@ -416,13 +417,26 @@ fn concat_loop(strings: List(String), accumulator: String) -> String {
 /// ```
 ///
 pub fn repeat(string: String, times times: Int) -> String {
-  repeat_loop(string, times, "")
+  case times <= 0 {
+    True -> ""
+    False -> repeat_loop(string, times, string, "")
+  }
 }
 
-fn repeat_loop(string: String, times: Int, acc: String) -> String {
+fn repeat_loop(
+  string: String,
+  times: Int,
+  doubling_acc: String,
+  acc: String,
+) -> String {
+  let acc = case int.bitwise_and(times, 1) {
+    0 -> acc
+    _ -> acc <> doubling_acc
+  }
+  let times = int.bitwise_shift_right(times, 1)
```

Comment:
I'd use `times / 2`, it's easier to understand and a lot faster on the JS target

Reply:
Nice! Back to no longer needing `int`!